clients/java/dkv-client/src/test/java/org/dkv/client/SimpleDKVClientTest.java

```diff
@@ -98,6 +98,29 @@ public void shouldPerformAtomicIncrDecr() throws InterruptedException {
         assertEquals(0L, convertToLong(actualBts));
     }
 
+    @Test
+    public void shouldPerformPutTTLAndGet() throws InterruptedException {
+        String key = "helloTTL", expVal = "world";
+        Long expiryTS = (System.currentTimeMillis() / 1000) + 2;
+        dkvCli.put(key, expVal, expiryTS);
+        String actVal = dkvCli.get(Api.ReadConsistency.LINEARIZABLE, key);
+        assertEquals(format("Invalid value for key: %s", key), expVal, actVal);
+        Thread.sleep(3000);
```

Comment:
Using the technique used in the server side tests, check if this `Thread.sleep()` can be avoided in favor of fast tests.